packages/node/src/yargs.ts

```diff
@@ -29,6 +29,11 @@ export function getYargsOption() {
       demandOption: false,
       describe: 'Use local mode',
     },
+    'force-clean': {
```

Comment:
We probably want a command so its like `subql-node force-clean --subquery-name=app`

Reply:
i'm ok with what it is now, basically means clean data and start indexing. If we make it a subcommand, it will only be used to clean data.

Reply:
good points, i was tossing up which way to implement but ended up going with how it is now because during development it's very likely that you're going to want to clean the project and then start indexing immediately (make changes to schema and then index and test on the playground)